grakn-graph/src/main/java/ai/grakn/factory/AbstractInternalFactory.java

```diff
@@ -34,6 +34,8 @@
     protected G graph = null;
     private G batchLoadingGraph = null;
 
+    private Boolean lastGraphBuiltBatchLoading = null;
```

Comment:
Setting it to false initially would mean that the last graph the factory built was a normal graph. Which at the very start is not true.